src/api.ts

```diff
@@ -77,7 +77,6 @@ export interface RoutesRequest {
 export interface RouteOptions {
   integrator?: string // Should contain the identifier of the integrator. Usually, it's dApp/company name.
   fee?: number // 0.03 = take 3% integrator fee (requires verified integrator to be set)
-  insurance?: boolean // Whether the user wants to insure their tx
```

Comment:
we can't do this, it will be a breaking change

Reply:
We will merge it and https://github.com/lifinance/lifi-backend/pull/5026 this one at the same time

Reply:
Possilbe progress: merge this types, and publish it, then update my pr on BE, finally merge my be pr, what you think

Reply:
fast and safe, I reckon

Reply:
I mean not for the BE but for the partners
We can't just remove some property from the request 😁

Reply:
ok, let's keep what the jira ticket said, refuse any request with insurance parameter, send an announcement to partners, and move it